src/main/java/mcjty/theoneprobe/items/TopArmorMaterial.java

```diff
@@ -13,17 +13,19 @@
     private final int[] damageReduction;
     private final int enchantability;
     private final float toughness;
+    private final float field_234660_o_;
```

Comment:
This and the parameter can just be called `knockbackResistance` or something rather than copying the srg name that shows for vanilla of `field_234660_o_`